### DISCUSSION TITLE
tutorial-extras/led

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#26] [comment]
# tutorial-extras/led

简介 本章介绍使用arduino烧录Air001开发板控制其上3个LED灯闪烁 硬件准备 按☁️ Air001开发板入门 (/tutorial-advanced/Air001_start.html)，将Air001和DAPLink调试器使用排针排母连接 在Air001开发板上，板载的三个LED灯，分别由PB1、`PB0和PB3三个GPIO控制 具体可参...

https://arduino.luatos.com/tutorial-extras/led.html

**Comment by lishuzhou** [comment]
Air001 看芯片手册是有片上 rtc 资源的，Arduino 如何开发呢，有没有相关 demo？
